### DISCUSSION TITLE
New requested machines onto 86box

### DISCUSSION THREAD

**Discussion by Tomeister1** [86Box/86Box#4023] [General]
Hi do you know when the new requested machines and system roms for 86box will be ready and merged into master roms? I am trying to wait and stay patient for them. But I don't want to give you guys trouble. Here is a look of the requested machines.

Thanks![image](https://github.com/86Box/86Box/assets/79490867/3779983c-76ee-4e6b-a509-71ab19fe524e)![image](https://github.com/86Box/86Box/assets/79490867/4c5cda6e-388e-4a4b-9cd0-51c10ce2599d)![image](https://github.com/86Box/86Box/assets/79490867/858e9e14-2149-4386-ba73-3c702a964259)

**Comment by jriwanek** [comment]
They'll be merged to master when they're ready, there are no timeframes, but asking us when x will be ready only slows us down as we have to give answers which take time.
